client/src/reducers/undoableConfig.js

```diff
@@ -29,7 +29,10 @@ const skipOnActions = new Set([
   "clear all user defined genes",
 
   "get single gene expression for coloring started",
-  "get single gene expression for coloring error"
+  "get single gene expression for coloring error",
+
```

Comment:
Whitespace

Reply:
I might be looking at the wrong this, but I believe the whitespace here is purposeful, to separate the actions of each reducer.

Reply:
Roger. Feel free to run prettier and let it decide.